File: utils/validator.js
function isValidEmail(email) {
  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
  return emailRegex.test(email);
}

function isStrongPassword(password) {
  return password && password.length >= 6;
}

function sanitizeEmail(email) {
  return email ? email.trim() : "";
}

module.exports = { isValidEmail, isStrongPassword, sanitizeEmail };

File: utils/validator.py
import re

def is_valid_email(email):
    return bool(re.match(r'^[^\s@]+@[^\s@]+\.[^\s@]+$', email))

def is_strong_password(password):
    return bool(password and len(password) >= 6)

def sanitize_email(email):
    return email.strip() if email else ""
